internal/service/ses/ses_cluster.go

```diff
@@ -341,7 +341,7 @@ func resourceNcloudSESClusterRead(ctx context.Context, d *schema.ResourceData, m
 		return diag.FromErr(err)
 	}
 
-	if cluster == nil {
+	if cluster == nil || *cluster.ClusterStatus == SESStatusReturnCode {
```

Comment:
It looks like you are introducing another reason for the change, could you split this change as a separate PR?